### DISCUSSION TITLE
CalDav integration

### DISCUSSION THREAD

**Discussion by Cmhorn** [stacks-task-manager/stacks2#4] [Ideas & Feature requests]
Just curious where this landed. I don't see anything obvious in the beta. 

Any update appreciated.

**Comment by synapse** [comment]
Hey @Cmhorn,

I didn't had the time to get to it just yet but it is in **todo**.

**Reply by tokeriis** [reply]
I don't see CalDAV integration on the todo list in the roadmap, is it still planned? I would love to at least be able to subscribe to calendars outside Stacks.
